components/gpio_control/gpio_control.py

```diff
@@ -90,7 +90,7 @@ def get_all_devices(config):
     logger.setLevel('INFO')
 
     config = configparser.ConfigParser(inline_comment_prefixes=";")
-    config_path = os.path.expanduser('~/.config/phoniebox/gpio_settings.ini')
```

Comment:
There are suggestions (https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/774#issuecomment-626276063, #797) to move config files and content out our the phoniebox directory, but I don’t think there’s a decision yet.
@MiczFlor, what’s your opinion on this?

Nevertheless for the moment I wouldn’t change the current location, because this may break existing installations.

Reply:
As you like. There are definitely up's and down's to it.
For downwards compatiblity I could also include a check to see if there is a config file at `~/.config/phoniebox/` and copy it into the settings-folder.

Reply:
Ok I read up on the linked discussions. Wouldn't it be better to keep the settings in line with the rest of the settings by now and update it when a consensus about a new format is found?

Reply:
I agree, but people are already using GPIO control and we should ensure their installations don’t break when upgrading.

Reply:
Hi @themorlan ,
I am not a fan of mixing installation with configuration as this usually result in issues when one reinstalls the scripts. 
There are also many settings which are set outside of the RPi-Jukebox-RFID folder like all settings of mopidy, mpd etc which are typically inside the ~/.config folder.
I dont see the advatnage to have it inside the installation...

Reply:
@s-martin @veloxidSchweiz I see your points.

Reply:
I reverted the location of the gpio_settings.ini back to ~/.config/phoniebox/.
While I was at it, I grouped most of the lines which belong to GPIO-install into one block.